YamlSplitter/Program.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Tools.YamlSplitter
+{
+    using Microsoft.DocAsCode.Build.SchemaDriven;
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Tools.YamlSplitter.Models;
+
+    using CommandLine;
+    using Newtonsoft.Json.Schema;
+    using YamlDotNet.RepresentationModel;
+
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text;
+
+    class Program
+    {
+        static HashSet<string> missingMergeKey = new HashSet<string>();
+
+        static void Main(string[] args)
+        {
+            CommandLine.Parser.Default.ParseArguments<InitOptions, UpdateOptions>(args)
+                .MapResult(
+                (InitOptions opts) => { Console.WriteLine("init mode is not supported right now."); return 0; },
+                (UpdateOptions opts) => RunUpdate(opts),
+                errs => -1);
+            foreach(var key in missingMergeKey)
+            {
+                Console.WriteLine(key + " does not have a merge key defined in schema, skipping...");
+            }
+        }
+
+        public static int RunUpdate(UpdateOptions opt)
+        {
+            opt.InputYamlPath = Path.GetFullPath(opt.InputYamlPath);
+            if (!string.IsNullOrEmpty(opt.MDPath))
+            {
+                opt.MDPath = Path.GetFullPath(opt.MDPath);
+            }
+            if (!string.IsNullOrEmpty(opt.OutputYamlPath))
+            {
+                opt.OutputYamlPath = Path.GetFullPath(opt.OutputYamlPath);
+            }
+            var schemas = LoadSchemas(opt.SchemaFolder);
+            foreach (var ymlFile in Directory.EnumerateFiles(opt.InputYamlPath, "*.yml", SearchOption.AllDirectories))
+            {
+                if (ymlFile.ToLower().EndsWith("/toc.yml"))
```

Comment:
use this one `bool EndsWith(String value, StringComparison comparisonType)`

Reply:
done